routes/nodebb.js

```diff
@@ -112,7 +116,7 @@ app.post(`${BASE_REPORT_URL}/v2/users/:uid/tokens`, proxyObject());
 app.delete(`${BASE_REPORT_URL}/v2/users/:uid/tokens/:token`, proxyObject());
 app.get(`${BASE_REPORT_URL}/user/username/:username`, proxyObject());
 
-app.post(`${BASE_REPORT_URL}/user/v1/create`, proxyObject());
+app.post(`${BASE_REPORT_URL}/user/v1/create`, bodyParser.json({ limit: '10mb' }), proxyObject());
```

Comment:
As I mentioned create a different handler for this logic. Presently 138 line code changes will affect all other routes as well.

Reply:
created